orchestrator/__init__.py

```diff
@@ -10,46 +10,28 @@
 from dagster_aws.s3 import S3Resource
 from dagster_aws.pipes import PipesLambdaClient
 
-from orchestrator.assets.postgres import (
-    mitos_dbt_assets,
-    output_test_asset,
-)
-from orchestrator.assets.business_travel import (
-    travel_spending,
-    annual_cpi_index,
-    cost_object_warehouse,
-    expense_category_mapper,
-    expense_emission_mapper,
-    mode_co2_mapper,
-    all_scope_summary,
-    dhub_travel_spending,
-)
+from orchestrator.assets.postgres import mitos_dbt_assets
+from orchestrator.assets import construction, business_travel
+
+from orchestrator.jobs.business_travel_job import business_asset_job
+from orchestrator.jobs.construction_job import construction_asset_job
+
 from orchestrator.constants import (
     dbt_project_dir,
     DWRHS_CREDENTIALS,
     PG_CREDENTIALS,
     dh_api_key,
 )
-from orchestrator.jobs.business_travel_job import business_asset_job
 from orchestrator.resources.datahub import DataHubResource
 from orchestrator.resources.mit_warehouse import MITWHRSResource
 from orchestrator.schedules.mitos_warehouse import schedules
 
+construction_assets = load_assets_from_modules([construction])
```

Comment:
Key change here: now use `load_assets_from_modules` to load and parse all assets from corresponding file.